pyvo/registry/regtap.py

```diff
@@ -551,6 +554,25 @@ def creators(self):
         """
         return self.get("creator_seq", default="", decode=True).split(";")
 
+    @property
+    def created(self):
+        """Date of creation of the resource."""
+        return self.get("created", decode=True)
+
+    @property
+    def updated(self):
+        """Date of last modification of the resource."""
+        return self.get("updated", decode=True)
+
+    @property
+    def rights(self):
+        """Rights information about the content of the resource.
```

Comment:
VOResource says on "rights": "A statement of usage conditions", which I think I like better than "Rights information".